docs/source/app-dev/bindings-java/quickstart.rst

```diff
@@ -356,124 +356,126 @@ The *Issuers* of the two Ious, which are involved in the transaction because the
+A script test is defined with ``trade_test = script do``. The ``submit`` function takes a submitting party and a transaction, which is specified the same way as in contract choices.
 
 The following block, for example, issues an ``Iou`` and transfers it to Alice:
 
 .. literalinclude:: quickstart/template-root/daml/Tests/Trade.daml
   :language: daml
-  :start-after: -- BEGIN_SCENARIO
-  :end-before: -- END_SCENARIO
+  :start-after: -- BEGIN_SCRIPT
+  :end-before: -- END_SCRIPT
 
-Compare the scenario with the ``setup`` scenario in ``daml/Main.daml``. You will see that the scenario you used to initialize the sandbox is an initial segment of the ``trade_test`` scenario. The latter adds transactions to perform the trade you performed through Navigator, and a couple of transactions in which expectations are verified.
+Compare the script with the ``initialize`` script in ``daml/Main.daml``. You will see that the script you used to initialize the sandbox is an initial segment of the ``trade_test`` script. The latter adds transactions to perform the trade you performed through Navigator, and a couple of transactions in which expectations are verified.
 
-After a short time, the text *Scenario results* should appear above the test. Click on it to open the visualization of the resulting ledger state.
+After a short time, the text *Script results* should appear above the test. Click on it to open the visualization of the resulting ledger state.
 
 .. figure:: quickstart/images/ledger.png
 
 Each row shows a contract on the ledger. The first four columns show which parties know of which contracts. The remaining columns show the data on the contracts. You can see past contracts by checking the **Show archived** box at the top. Click the adjacent **Show transaction view** button to switch to a view of the entire transaction tree.
 
-In the transaction view, transaction ``#6`` is of particular interest, as it shows how the Ious are exchanged atomically in one transaction. The lines starting ``known to (since)`` show that the Banks do indeed not know anything they should not:
+In the transaction view, transaction ``6`` is of particular interest, as it shows how the Ious are exchanged atomically in one transaction. The lines starting ``known to (since)`` show that the Banks do indeed not know anything they should not:
 
 .. code-block:: none
 
-  TX #6 1970-01-01T00:00:00Z (Tests.Trade:61:14)
+  TX 6 1970-01-01T00:00:00Z (Tests.Trade:70:14)
   #6:0
-  │   known to (since): 'Alice' (#6), 'Bob' (#6)
+  │   known to (since): 'Alice' (6), 'Bob' (6)
   └─> 'Bob' exercises IouTrade_Accept on #5:0 (IouTrade:IouTrade)
             with
               quoteIouCid = #3:1
       children:
       #6:1
-      │   known to (since): 'Alice' (#6), 'Bob' (#6)
+      │   known to (since): 'Alice' (6), 'Bob' (6), 'EUR_Bank' (6)
```

Comment:
interestingly, it seems that at some point something changed so that now `EUR_Bank` knows about this contract quite a bit earlier (compare to `#6:7` in the previous output), and `USD_Bank` knows about it from `#6:2` rather than `#6:3`. I'm not sure what the implications of this are for the statement above,

> The lines starting ``known to (since)`` show that the Banks do indeed not know anything they should not